### DISCUSSION TITLE
Dubbed anime with Captions or Subtitles

### DISCUSSION THREAD

**Discussion by ajayanuragi** [aniyomiorg/aniyomi#848] [General]
In zoro.to website there is option of having subtitles even with dubbed anime. Ig it will be cool if we can have same option in aniyomi app.

**Comment by Samfun75** [comment]
Update to latest beta

**Reply by Kakarot2005** [reply]
How can I update to latest beta?

**Reply by ajayanuragi** [reply]
From here 

https://github.com/jmir1/aniyomi-mpv-beta
